process/economics/economicsData.go

```diff
@@ -13,41 +13,46 @@ import (
 	"github.com/ElrondNetwork/elrond-go/process"
 )
 
-var _ process.RewardsHandler = (*EconomicsData)(nil)
-var _ process.FeeHandler = (*EconomicsData)(nil)
+var _ process.EconomicsDataHandler = (*data)(nil)
+var _ process.RewardsHandler = (*data)(nil)
+var _ process.FeeHandler = (*data)(nil)
 
+var epsilon = 0.00000001
 var log = logger.GetOrCreate("process/economics")
 
-// EconomicsData will store information about economics
-type EconomicsData struct {
+// data will store information about economics
+type data struct {
```

Comment:
economics would have been better, data is too general

Reply:
conflict with the package name

Reply:
made economicsData